server/src/events.mjs

```diff
@@ -160,28 +176,28 @@ function coreSendEvent(isBroadcast, ws, userId, method, result, msg, fSuccess, f
               set: function ss(key, val) { return stateManagement.setScratch(userId, key, val) },
               get: function gs(key) { return stateManagement.getScratch(userId, key); },
               sendEvent: function(method, result, msg) {
-                function fSuccess() {
-                  logger.info(`${msg}: Sent event ${method} with result ${JSON.stringify(result)}`)
-                }
-                function fErr() {
-                  logger.info(`Could not send ${method} event because no listener is active`)
-                }
-                function fFatalErr() {
-                  logger.info(`Internal error`)
-                }
-                sendEvent(ws, userId, method, result, msg, fSuccess, fErr, fFatalErr);
+                sendEvent(
```

Comment:
What's the reason for splitting this up into multiple lines?

Reply:
Taken care